strategy/sim2.py

```diff
@@ -30,10 +31,36 @@ def __init__(self, name, team, world, speed, location):
         self.speed = speed
         self.location = location
         self.has_cube = True
-
+    def Robot1(drive):
+        Robot.name = 'Robot#1'
+        Robot1.team = BLUE
+        Robot1.speed = 5
+        Robot1.location = BLUE_PORTAL
+        Robot1.has_cube = True
+        
+        
+
+        if has_cube == False:
+            Robot1.pick_up
+        else:     
+            Robot1.put_down
     def __lt__(self, other):
         return hash(self) < hash(other)
-
+    def world():
+        if time < 150:
+            return BLUE_PILE, BLUE_SWITCH, BLUE_SCALE 
+            return RED_PILE, RED_SWITCH, RED_SCALE
+            if BLUE_SWITCH == True:
+                BLUE_SCORE += 1
+            if RED_SWITCH == True:
+                RED_SCORE += 1
+            if RED_SCALE > BLUE_SCALE:
+                RED_SCORE += 1
+            if BLUE_SCALE > RED_SCALE:
+                BLUE_SCORE += 1
+            time += 1    
```

Comment:
We don't need to keep track of time, the scheduler function will do that for us. (We could make it available on `world` if we want?)